cmd/cloud/deployment.go

```diff
@@ -325,6 +334,64 @@ func newDeploymentVariableUpdateCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+//nolint:dupl
+func newDeploymentHibernateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "hibernate [DEPLOYMENT-ID]",
+		Aliases: []string{"hb"},
+		Short:   "Hibernate an Astro Deployment",
+		Long:    "Hibernate an Astro Deployment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if resumeSchedules {
+				return deploymentOverrideHibernation(cmd, args, nil, nil)
+			}
+
+			hibernating := true
+			overrideUntil, err := getOverrideUntil(until, forDuration)
+			if err != nil {
+				return err
+			}
+			return deploymentOverrideHibernation(cmd, args, &hibernating, overrideUntil)
+		},
+	}
+	cmd.Flags().StringVarP(&deploymentName, "deployment-name", "n", "", "Name of the deployment to hibernate")
+	cmd.Flags().StringVarP(&until, "until", "u", "", "Time to hibernate until")
+	cmd.Flags().StringVarP(&forDuration, "for", "d", "", "Time to hibernate for")
+	cmd.Flags().BoolVarP(&resumeSchedules, "resume-schedules", "r", false, "Remove the existing override and resume schedules")
+	cmd.Flags().BoolVarP(&forceOverride, "force", "f", false, "Force hibernate: Don't prompt a user before Deployment hibernation")
+	cmd.MarkFlagsMutuallyExclusive("until", "for", "resume-schedules")
+	return cmd
+}
+
+//nolint:dupl
+func newDeploymentWakeUpCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "wake-up [DEPLOYMENT-ID]",
+		Aliases: []string{"wu"},
+		Short:   "Wake up an Astro Deployment",
+		Long:    "Wake up an Astro Deployment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if resumeSchedules {
+				return deploymentOverrideHibernation(cmd, args, nil, nil)
+			}
+
+			hibernating := false
+			overrideUntil, err := getOverrideUntil(until, forDuration)
+			if err != nil {
+				return err
+			}
+			return deploymentOverrideHibernation(cmd, args, &hibernating, overrideUntil)
+		},
+	}
+	cmd.Flags().StringVarP(&deploymentName, "deployment-name", "n", "", "Name of the deployment to wake up")
+	cmd.Flags().StringVarP(&until, "until", "u", "", "Time to wake up until")
+	cmd.Flags().StringVarP(&forDuration, "for", "d", "", "Time to wake up for")
```

Comment:
nit: might be worth mentioning the expected time string format(s) in the flag description, so that users don't end up trying out all sorts of permutations 😅

Reply:
Totally!